src/views/project/institutions/ListView.vue

```diff
@@ -0,0 +1,208 @@
+const projectId = route.params.id
+const ProjectInstitutionsStore = useProjectInstitutionStore()
+const institutionsToDelete = ref([])
+
+const isLoaded = computed(() => ProjectInstitutionsStore.isLoaded)
+
+const selectedLetter = ref(null)
+const letters = computed(() => {
+  const letters = new Set()
+  for (const institution of ProjectInstitutionsStore.institutions) {
+    if (institution?.name?.length > 0) {
+      const firstLetter = institution.name[0]
+      letters.add(firstLetter.toUpperCase())
+    }
+  }
+  return [...letters].sort()
+})
+
+const filters = reactive({})
+const filteredInstitutions = computed(() =>
+  Object.values(filters)
+    .reduce(
+      (institutions, filter) => institutions.filter(filter),
+      ProjectInstitutionsStore.institutions
+    )
+    .sort((a, b) => {
+      const nameA = a.name
+      if (!nameA) {
+        return -1
+      }
+
+      const nameB = b.name
+      if (!nameB) {
+        return -1
+      }
+
+      const compare = nameA.localeCompare(nameB)
+      if (compare) {
+        return compare
+      }
+    })
+)
+
+const allSelected = computed({
+  get: function () {
+    return filteredInstitutions.value.every((b) => b.selected)
+  },
+  set: function (value) {
+    filteredInstitutions.value.forEach((b) => {
+      b.selected = value
+    })
+  },
+})
+const someSelected = computed(() =>
+  filteredInstitutions.value.some((b) => b.selected)
+)
+
+onMounted(() => {
+  if (!ProjectInstitutionsStore.isLoaded) {
+    ProjectInstitutionsStore.fetchInstitutions(projectId)
+  }
+})
+
+function refresh() {
+  ProjectInstitutionsStore.fetchInstitutions(projectId)
+}
+
+function setPage(event) {
+  const text = event.target.textContent
+  if (text == 'ALL') {
+    clearFilters()
+  } else {
+    filterByLetter(text)
+  }
+}
+
+function clearFilters() {
+  selectedLetter.value = null
+  delete filters['page']
+}
+
+function filterByLetter(letter) {
+  selectedLetter.value = letter
+  filters['page'] = (institution) => {
+    if (institution?.name.length > 0) {
+      const name = institution?.name[0]
+      return name.toUpperCase() == letter
+    }
+    return false
+  }
+}
+</script>
+
+<template>
+  <LoadingIndicator :isLoaded="isLoaded">
+    <h1>Project Institutions</h1>
+
+    <header>
+      There are
+      {{
+        ProjectInstitutionsStore.institutions.length != 0
+          ? ProjectInstitutionsStore.institutions.length
+          : 'no'
+      }}
+      institutions associated with this project.
+    </header>
+
+    <div class="action-bar">
+      <RouterLink :to="`/myProjects/${projectId}/institutions/create`">
+        <button type="button" class="btn btn-m btn-outline-primary">
+          <i class="fa fa-plus"></i>
+          <span>Add Institutions</span>
+        </button>
+      </RouterLink>
+    </div>
+
+    <div v-if="filteredInstitutions.length">
+      <div class="alphabet-bar">
+        Display institutions beginning with:
+        <template v-for="letter in letters">
```

Comment:
We don't need a letters bar here. Projects should have a limited number of institutions so they don't benefit from filtering by name. We can just list all the projects. Can you delete this logic?